src/components/Login.js

```diff
@@ -95,6 +96,10 @@ class Login extends Component {
                         AccessToken.getCurrentAccessToken().then(
                           (data) => {
                             alert(data.accessToken.toString());
```

Comment:
This alert thing is only for debug purposes, right? Please add a TODO to remove this before we go production.